ExcelMapper/ExcelMapper.cs

```diff
@@ -358,6 +434,91 @@ IEnumerable Fetch(ISheet sheet, Type type)
             }
         }
 
+        IEnumerable<dynamic> FetchDynamic(ISheet sheet)
```

Comment:
This duplication should be avoided. Perhaps we can create custom `ColumnInfo` objects for the dynamic case in `GetColumnInfo` and eliminate this method.

Reply:
`ColumnInfo` is stored at "Type" level in `TypeMapperFactory`. For dynamic there is no prior `Type`.

Reply:
> 
> 
> What about saving? Doesn't this cause saving tracked objects to fail?

We can't add a `SaveDynamic(IEnumerable<dynamic> rows, string filepath)` that reproduce the file identically.
Its useless (ExcelMapper vs File.Copy).

we can add a `SaveDynamic(IEnumerable<dynamic> rows, string filepath, IEnumerable<ColumInfo>)` filling the gap of lacking Type. The dev have to provide its saving contract _dynamicaly_.
But in my opinion, it's usless.

It's more compliant with the current Api to do as do in my `FetchDynamicTest`.
Use a real `Type` that will gives you control over the saved Excel document using `[Column]` and stuff and invoke the current `Save()` methods available.

I'm not sure about the tracked objects mechanics.

Reply:
> 
> > What about saving? Doesn't this cause saving tracked objects to fail?
> 
> We can't add a `SaveDynamic(IEnumerable<dynamic> rows, string filepath)` that reproduce the file identically.
> Its useless (ExcelMapper vs File.Copy).

The only use i can see for symmetrical format save is to filter/remove/add rows before saving.
We could have a context
```
// Saving dynamics
SaveDynamic(IEnumerable<dynamic> rows, string filepath, SaveDynamicContext context);
// Saving context
public class SaveDynamicContext {
    string SheetName;
    int SheetIndex;
    bool IncludeHeaderRow;
    Predicate<dynamic> RowFilter;
    string[] ExcludedColumn;
    Dictionary<int, string> ColumOrder;// if null try to keep column order from source file
}
```

Reply:
Something neat would be to dispatch IEnumerable<dynamic> to multiple Sheet in the saved document with multiple format and filtering logic.
```
// Signature
SaveDynamic(IEnumerable<dynamic> rows, string filepath, param SaveDynamicContext context);
// e.g
SaveDynamic(dynaRows, "myresult.xlsx"
    , new SaveDynamicContext { // Sheet 1
        SheetName = "Greater than 1",
        SheetIndex = 1, // or 0 whatever index base it is
        IncludeHeaderRow = true,
        RowFilter = r => r.Price > 1 && r.Value > 1 ,
        ExcludedColumn = new[]{ "NumberInStock", "OfferEnd" }
}   , new SaveDynamicContext { // Sheet 2
        SheetName = "Offers Only",
        SheetIndex = 2,
        IncludeHeaderRow = false,// No headers
        RowFilter = r => r.Offer == true ,
        ExcludedColumn = new[]{ "NumberInStock", "OfferEnd" }
});
```